src/commands/publish.ts

```diff
@@ -0,0 +1,40 @@
+import { Command } from '@oclif/core'
+import { BundleDescriptorService } from '../services/bundle-descriptor-service'
+import {
+  ConfigService,
+  DOCKER_ORGANIZATION_PROPERTY
+} from '../services/config-service'
+import { DockerService } from '../services/docker-service'
+
+export default class Publish extends Command {
+  static description = 'Publish bundle Docker images'
```

Comment:
![image](https://user-images.githubusercontent.com/1956696/173298238-a55c8e4c-b6a9-4f1a-9cd8-d9e4281c237a.png)
I would rename image -> images in pack command as well